### PR TITLE
Added new UI test target and added tests for Atome and Affirm component

### PR DESCRIPTION
**Pull request by neelSharma12**
This PR contains following things: 
- Separate target for UI tests. 
- Atome component end to end UI tests
- UI tests of Affirm component are moved from the UI test target.

### CODE REVIEW

File: AdyenUIHostUITests/Components/Affirm/AffirmComponentUITests.swift [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+            XCTAssertEqual(details.telephoneNumber, "2025550146")
+            XCTAssertEqual(details.emailAddress, "katrina@mail.com")
+            XCTAssertEqual(details.billingAddress, expectedBillingAddress)
+            XCTAssertEqual(details.deliveryAddress, expectedDeliveryAddress)
+            
+            sut.stopLoadingIfNeeded()
+            didSubmitExpectation.fulfill()
+        }
+
+        wait(for: .seconds(1))
```
Comment (erenbesel) [inline AdyenUIHostUITests/Components/Affirm/AffirmComponentUITests.swift]: We recently reduced this number to 300 miliseconds on other tests. If 1 sec isn't necessary here, let's align with 300 ms.

File: AdyenUIHostUITests/Components/AtomeComponentUITests.swift [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+        houseNumberTextfield.tap()
+        houseNumberTextfield.typeText("345")
+
+        let postalCodeTextfield = app.otherElements.textFields[AtomeViewIdentifier.postalCode]
+        postalCodeTextfield.tap()
+        postalCodeTextfield.typeText("1089")
+
+        app.otherElements.buttons[AtomeViewIdentifier.payButton].tap()
+
+        wait(for: .seconds(5))
```
Comment (erenbesel) [inline AdyenUIHostUITests/Components/AtomeComponentUITests.swift]: These durations (5 sec, 2 sec) seem very long but I don't know much about UI tests. Are they needed to be this much in UI tests?

File: AdyenUIHostUITests/Components/Affirm/AffirmComponentUITests.swift (hunk 1 above)
Comment (neelSharma12) [reply AdyenUIHostUITests/Components/Affirm/AffirmComponentUITests.swift]: This I have modified.

File: AdyenUIHostUITests/Components/Affirm/AffirmComponentUITests.swift [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+    private enum AffirmViewIdentifier {
+        static let firstName = "AdyenComponents.AffirmComponent.firstNameItem"
+        static let lastName = "AdyenComponents.AffirmComponent.lastNameItem"
+        static let phone = "AdyenComponents.AffirmComponent.phoneNumberItem"
+        static let email = "AdyenComponents.AffirmComponent.emailItem"
+        static let billingAddress = "AdyenComponents.AffirmComponent.addressItem"
+        static let deliveryAddress = "AdyenComponents.AffirmComponent.deliveryAddressItem"
+        static let deliveryAddressToggle = "AdyenComponents.AffirmComponent.deliveryAddressToggleItem"
+        static let payButton = "AdyenComponents.AffirmComponent.payButtonItem.button"
+    }
```
Comment (descorp) [inline AdyenUIHostUITests/Components/Affirm/AffirmComponentUITests.swift:241]: This is nice structure 👍 
We should adopt this more

**Review by erenbesel — DISMISSED** [review]

**Review by descorp — DISMISSED** [review]

File: Demo/Configuration.swift [hunk 4]
```diff
@@ -83,9 +83,17 @@ internal struct Configuration: Codable {
     )
     
     fileprivate static func loadConfiguration() -> Configuration {
-        UserDefaults.standard.data(forKey: defaultsKey)
-            .flatMap { try? JSONDecoder().decode(Configuration.self, from: $0) }
-            ?? defaultConfiguration
+        if CommandLine.arguments.contains("SY") {
+           return Configuration(countryCode: "SY",
```
Comment (descorp) [inline Demo/Configuration.swift]: We can fix it another PR:

Maybe we can use `countryCode` and `currencyCode` as variables ?
If tomorrow we decide to write tests for Blik(LPM from Poland) - do we need to add 
`if CommandLine.arguments.contains("PO") { return Configuration(...) }` ?

**Review by nauaros — DISMISSED** [review]

**Review by erenbesel — APPROVED** [review]

**Review by descorp — APPROVED** [review]

**Pull request merged**